src/styles/_catalog.scss

```
@@ -1,22 +1,34 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
-  max-width: 944px;
-  margin: 0 auto;
+  display: grid;
+  grid-template-columns: repeat(auto-fill,200px);
+  gap: 48px;
+  justify-content: center;
 
   &__card {
-    display: flex;
+    display: grid;
     justify-content: center;
     align-items: center;
     width: 200px;
```

```suggestion
    width: 200px;
```
you should create variable for this value, cause you use it a lot in media queries